cime_config/acme/machines/Makefile

```diff
@@ -846,11 +804,12 @@ clean: cleanatm cleanocn cleanwav cleanglc cleanice cleanrof cleanlnd
 
 realclean: clean cleancsmshare cleanpio cleanmct cleangptl
 
+
 # the if-tests prevent DEPS files from being created when they're not needed
 ifneq ($(MAKECMDGOALS), db_files)
 ifneq ($(MAKECMDGOALS), db_flags)
 ifeq (,$(findstring clean,$(MAKECMDGOALS)))
-    -include $(CURDIR)/Depends $(CASEROOT)/Depends.$(COMPILER) $(CASEROOT)/Depends.$(MACH) $(CASEROOT)/Depends.$(MACH).$(COMPILER)
```

Comment:
Don't delete the $(MACH).$(COMPILER).  we added that to cope with multiple compilers on Titan that all needed something different.